packages/renderer/src/lib/ui/Button.svelte

```diff
@@ -21,7 +21,7 @@ export let padding: string =
 let iconType: string | undefined = undefined;
 
 onMount(() => {
-  if (icon?.prefix === 'fas') {
+  if (['fas', 'fab'].includes(icon?.prefix)) {
```

Comment:
I think we have some code with 'fa' as prefix so maybe would be simpler

if you want to cover all icons then you'll be missing regular (`far` and maybe more)
https://github.com/FortAwesome/Font-Awesome/blob/6.x/js-packages/%40fortawesome/free-regular-svg-icons/faAddressCard.js#L3

Reply:
So what you recommend is doing like `icon?.prefix.startsWith('fa')` ?

Reply:
yes

Reply:
or maybe here we should have an unified utility class that really do checks all the available font awesome options

b/c I think each component is doing its own check

Reply:
Looks like we're doing this in at least 4 places, agreed a util function would be useful.

Reply:
I created a  `IconUtils` file to handle the logic, and changed the 4 places where the same logic was applied